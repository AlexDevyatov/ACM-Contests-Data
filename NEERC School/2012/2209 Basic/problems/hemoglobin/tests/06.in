50
+878
+536
?1
+733
+923
?4
?1
+710
+622
?4
+796
?4
?2
+444
?7
?1
?7
?5
+702
?3
?7
?2
+151
+776
+629
+653
?7
+795
+784
?15
?5
+5
?15
+980
?4
?2
+990
?13
+308
+620
+40
?5
+1
?16
+265
+464
?23
+706
+138
+22
