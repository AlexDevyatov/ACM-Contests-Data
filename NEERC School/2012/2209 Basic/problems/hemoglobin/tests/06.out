536
3070
923
2988
3051
1418
4764
444
4764
3495
1942
4930
1146
4151
10132
3637
9259
2564
985
8327
2938
8674
12927
